src/eei.cpp

```diff
@@ -619,7 +619,7 @@ inline int64_t maxCallGas(int64_t gas) {
 
       result.isRevert = import->base == Name("revert");
 
-      return Literal();
+      throw new EndExecution{};
```

Comment:
Should be `throw EndExecution{}`. You want to throw an exception of type `EndExecution` not `EndExecution*`.